Formula/vgmstream.rb

```diff
@@ -1,7 +1,7 @@
 class Vgmstream < Formula
   desc "Library for playing streamed audio formats from video games"
   homepage "https://hcs64.com/vgmstream.html"
-  url "http://svn.code.sf.net/p/vgmstream/code", :revision => 1040
+  url "https://gitlab.kode54.net/kode54/vgmstream.git", :tag => "r1040"
```

Comment:
:tag requires :revision, but you can actually use a tarball here.
https://gitlab.kode54.net/kode54/vgmstream/repository/archive.tar.gz?ref=r1040
or
https://gitlab.kode54.net/kode54/vgmstream/repository/archive.tar.bz2?ref=r1040